data/successors/successor jobs.txt

```diff
@@ -0,0 +1,788 @@
+	deadline
+	passengers 2 5 .5
+	to offer
+		has "known to the successors"
+		random < 20
+	source
+		attributes "successor"
+		not attributes "crime"
+	destination
+		attributes "successor"
+		not attributes "uninhabited"
+		attributes "education"
+		distance 5 20
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 22000 200
+		dialog phrase "successor students out"
+		"successor jobs" ++
+
+
+mission "Successor: Workers Out 1"
+	job
+	repeat
+	name "Successor workers to <planet>"
+	description "These <bunks> workers are looking for jobs on <destination> and will pay you <payment> to take them there by <date>."
+	deadline
+	passengers 2 5 .5
+	to offer
+		has "known to the successors"
+		random < 20
+	source
+		attributes "successor"
+	destination
+		attributes "successor"
+		not attributes "uninhabited" "quiet"
+		distance 5 20
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 22000 200
+		dialog phrase "successor workers out"
+		"successor jobs" ++
+
+
+mission "Successor: Workers Out 2"
+	job
+	repeat
+	name "Successor workers to <planet>"
+	description "These <bunks> workers are looking for jobs on <destination> and will pay you <payment> to take them there by <date>."
+	deadline
+	passengers 3 6 .5
+	to offer
+		has "known to the successors"
+		random < 20
+	source
+		attributes "successor"
+	destination
+		attributes "successor"
+		not attributes "uninhabited" "quiet"
+		distance 5 20
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 21500 200
+		dialog phrase "successor workers out"
+		"successor jobs" ++
+
+
+mission "Successor: Delivery Out 1"
+	job
+	repeat
+	name "Shipment to <planet>"
+	description "A courier is needed for <tons> of supplies from <origin> to <planet>. You will be paid <payment> on delivery."
+	cargo random 5 10 .2
+	to offer
+		has "known to the successors"
+		random < 35
+	source
+		attributes "successor"
+	destination
+		attributes "successor"
+		not attributes "uninhabited"
+		distance 3 20
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment
+		payment 2000 200
+		dialog phrase "successor delivery"
+		"successor jobs" ++
+
+
+mission "Successor: Delivery Out 2"
+	job
+	repeat
+	name "Shipment to <planet>"
+	description "A courier is needed for <tons> of supplies from <origin> to <planet>. You will be paid <payment> on delivery."
+	cargo random 6 9 .2
+	to offer
+		has "known to the successors"
+		random < 30
+	source
+		attributes "successor"
+	destination
+		attributes "successor"
+		not attributes "uninhabited"
+		distance 3 20
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment
+		payment 2000 200
+		dialog phrase "successor delivery"
+		"successor jobs" ++
+
+
+mission "Successor: High Value Delivery Out"
+	job
+	repeat
+	name "High-value shipment to <planet>"
```

Comment:
This seems like a dangerously long name. I would check to make sure it doesn't overflow when paired with the longest Successor planet name.